.github/workflows/ruby_tests.yml

```diff
@@ -4,94 +4,27 @@ on:
   push:
     branches:
     - master
-env:
-  RAILS_ENV: test
-  DATABASE_URL: postgresql://postgres:@localhost/test
-  DATABASE_CLEANER_ALLOW_REMOTE_DATABASE_URL: true
```

Comment:
Dropped this change, we already have this in reusable `workflows/foreman_plugin.yml`